File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L6/01/L6.py
import math

def hit_bmi(weight, height):
    return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

def main():
    weight = float(input("Enter weight in pounds: "))
    feet = float(input("Enter feet: "))
    inches = float(input("Enter inches: "))
    height = feet * 12 + inches

    bmi = hit_bmi(weight, height)
    print(f"BMI is {bmi:.2f}")

    if 18.5 <= bmi < 25:
        print("Normal")
    elif 25 <= bmi < 30:
        print("Overweight")
    elif bmi >= 30:
        print("Obese")
    else:
        print("Underweight")

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L6/01/L6.ts
import * as readline from 'readline';

function hitBmi(berat: number, tinggi: number): number {
    return berat * 0.45359237 / ((tinggi * 0.0254) * (tinggi * 0.0254));
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter weight in pounds: ', (weightInput) => {
    const berat = parseFloat(weightInput);

    rl.question('Enter feet: ', (feetInput) => {
        const feet = parseFloat(feetInput);
        rl.question('Enter inches: ', (inchesInput) => {
            const inci = parseFloat(inchesInput);
            const tinggi = feet * 12 + inci;

            const bmi = hitBmi(berat, tinggi);
            console.log(`BMI is ${bmi}`);

            if (bmi < 25 && bmi > 18.5) {
                console.log('Normal');
            } else if (bmi < 30 && bmi > 25) {
                console.log('Overweight');
            } else if (bmi > 30) {
                console.log('Obese');
            } else {
                console.log('Underweight');
            }

            rl.close();
        });
    });
});

